jenkins/release.jenkinsFile

```diff
@@ -14,4 +14,11 @@ standardReleasePipelineWithGenericTrigger(
             mavenArtifactsPath: "$WORKSPACE/repository/",
             autoPublish: true
         )
+        publishToArtifactsProdBucket(
+            assumedRoleName: 'sql-jdbc-upload-role',
+            source: "$WORKSPACE/libs/opensearch-sql-jdbc-shadow-*.jar",
+            destination: "https://artifacts.opensearch.org/opensearch-clients/jdbc/opensearch-sql-jdbc-shadow-${tag}.jar",
```

Comment:
```suggestion
            destination: "opensearch-clients/jdbc/opensearch-sql-jdbc-shadow-${tag}.jar",
```

Reply:
Thanks fixed in [19bdcc1](https://github.com/opensearch-project/sql-jdbc/pull/106/commits/19bdcc16f93f5420262134d23f80c3bc47077058)